AcceptApiChanges.ps1

```diff
@@ -4,15 +4,9 @@
 
 $ApprovalFiles = ".\Tests\Approval.Tests\ApprovedApi\FluentAssertions\";
 
-## Remove current "approved" API 
-Remove-Item "$ApprovalFiles\*.verified.txt"
```

Comment:
These changes fixes a problem, when only a subset of the files changes and hence only _some_ new .verified files are generated.

The changes aligns the behavior with `AcceptApiChanges.sh`